### DISCUSSION TITLE
Wich way to render 3D tiles for iTowns is the best?

### DISCUSSION THREAD

**Discussion by alexLuky** [iTowns/itowns#1880] [Q&A]
Hello, dear community!

We're trying to use the iTowns in our project but we faced the problem by rendering the 3D tiles sets. We have Bentley software and it renders the 3D tiles in Cesium way but these sets have pretty bad behavior in itowns. They're spherical and it looks like there are some issues with sse computation or culling test... We tried to recalculate them by region it becomes better but still has problems with permanent hiding and repainting the whole tile node. Maybe there is another software for rendering, perhaps Reality Capture or something else?

**Comment by jailln** [comment]
Hello,
Thanks for sharing your experience with rendering 3D Tiles with itowns. Could you share a tileset or a subset of your tileset that demonstrates the issues you're facing ? Could you also detail more precisely why you think there is sse computation or culling test issues ? (e.g. by describing the expected behavior vs the current behavior, and with small videos or screenshots)

**Reply by alexLuky** [reply]
@jailln 
I've made the issue for this previosly. You can take a [look](https://github.com/iTowns/itowns/issues/1859).
Here is the link to the 3D tiles rendered by Bentley from the box for Cesium, We have commercial responsibilties and we can't keep open free access to the tileset for a long time, so, please take a look as soon as possible. Here is the [link](https://storage.yandexcloud.net/gis-3d-models/data/tiles3d/Valaam_Aerial_Survey/Scene/Cesium.json).

**Comment by jailln** [comment]
I just found out that you already did [here](https://github.com/iTowns/itowns/issues/1859), sorry. I'll take a look at it :)
